phoenix-core/src/main/java/org/apache/phoenix/schema/DelegateTable.java

```diff
@@ -142,6 +143,14 @@ public Integer getBucketNum() {
         return delegate.getIndexes();
     }
 
+    @Override
+    public List<PTable> getIndexes(List<NamedNode> indexes) { return delegate.getIndexes(indexes); }
+
+    @Override
+    public List<PTable> getIndexes(boolean viewOnly) {
```

Comment:
Nit: Consistent indentation of the two functions. First one is all of one line, second one (though less wide) is broken down into multiple lines.